R/smtp_send.R

```diff
@@ -249,10 +249,8 @@ smtp_send <- function(email,
       `-to` = to,
       `-cc` = cc,
       `-bcc` = bcc,
-      `attach` = no_options(),
-      `-file` = tempfile_,
-      `-mime-type` = "text/html",
-      `-inline` = no_options()
+      `body` = no_options(),
+      `-file` = tempfile_
```

Comment:
```suggestion
      `-file` = tempfile_, 
      `-mime-type` = "text/html"
```

Reply:
@jcheng5 suggested that we make the mime-type explicit.

Reply:
> @jcheng5 suggested that we make the mime-type explicit.

Ok.